### PR TITLE
Custom Source: update Angular code snippets (T1149011)

### PR DESCRIPTION
**Pull request by sergepilipchuk**

### CODE REVIEW

File: concepts/70 Data Binding/00 Specify a Data Source/60 Custom Data Sources/2 Load Data/5 Server-Side Data Processing.md [hunk 1]
```diff
@@ -75,70 +75,79 @@ The following example shows a **CustomStore** that sends data processing setting
 ##### Angular
 
     <!-- tab: app.component.ts -->
-    import { Component } from '@angular/core';
     import { HttpClient, HttpParams } from '@angular/common/http';
-
+    import { LoadOptions } from 'devextreme/data';
     import CustomStore from 'devextreme/data/custom_store';
+    import { lastValueFrom } from 'rxjs';
+    import DataSource from 'devextreme/data/data_source';
```
Comment (artem-kurchenko) [inline concepts/70 Data Binding/00 Specify a Data Source/60 Custom Data Sources/2 Load Data/5 Server-Side Data Processing.md:82]: Let's re-group imports
  ```  
     import { Component } from '@angular/core';
     import { HttpClient, HttpParams } from '@angular/common/http';    
     import { lastValueFrom } from 'rxjs';
     
    import DataSource from 'devextreme/data/data_source';
    import CustomStore from 'devextreme/data/custom_store';    
    import { LoadOptions } from 'devextreme/data';
 ```

File: concepts/70 Data Binding/00 Specify a Data Source/60 Custom Data Sources/2 Load Data/5 Server-Side Data Processing.md [hunk 2]
```diff
@@ -75,70 +75,79 @@ The following example shows a **CustomStore** that sends data processing setting
...
 
     @Component({
         selector: 'app-root',
         templateUrl: './app.component.html',
         styleUrls: ['./app.component.css']
```
Comment (artem-kurchenko) [inline concepts/70 Data Binding/00 Specify a Data Source/60 Custom Data Sources/2 Load Data/5 Server-Side Data Processing.md]: Add providers
  providers:[HttpClient],

File: concepts/70 Data Binding/00 Specify a Data Source/60 Custom Data Sources/2 Load Data/5 Server-Side Data Processing.md [hunk 3]
```diff
@@ -75,70 +75,79 @@ The following example shows a **CustomStore** that sends data processing setting
...
+                            'userData',
+                        ].forEach(function (i) {
+                            if (i in loadOptions && isNotEmpty(loadOptions[i as keyof LoadOptions])) {
+                                params = params.set(i, JSON.stringify(loadOptions[i as keyof LoadOptions]));
+                            }
+                        });
+
+                        return lastValueFrom(
+                            this.http.get(
+                                'https://js.devexpress.com/Demos/WidgetsGalleryDataService/api/orders',
```
Comment (artem-kurchenko) [inline concepts/70 Data Binding/00 Specify a Data Source/60 Custom Data Sources/2 Load Data/5 Server-Side Data Processing.md]: return a fake link back: https://mydomain.com/MyDataService

File: concepts/70 Data Binding/00 Specify a Data Source/60 Custom Data Sources/2 Load Data/5 Server-Side Data Processing.md [hunk 4]
```diff
@@ -75,70 +75,79 @@ The following example shows a **CustomStore** that sends data processing setting
...
+                            'searchOperation',
+                            'searchValue',
+                            'select',
+                            'sort',
+                            'skip',
+                            'take',
+                            'totalSummary',
+                            'userData',
+                        ].forEach(function (i) {
+                            if (i in loadOptions && isNotEmpty(loadOptions[i as keyof LoadOptions])) {
```
Comment (artem-kurchenko) [inline concepts/70 Data Binding/00 Specify a Data Source/60 Custom Data Sources/2 Load Data/5 Server-Side Data Processing.md]: Small refactoring:

            const optionValue = loadOptions[i as keyof LoadOptions];
            if (i in loadOptions && isNotEmpty(optionValue)) {
              params = params.set(i, JSON.stringify(optionValue));
            }

Comment (sergepilipchuk) [reply concepts/70 Data Binding/00 Specify a Data Source/60 Custom Data Sources/2 Load Data/5 Server-Side Data Processing.md]: Done!

File: concepts/70 Data Binding/00 Specify a Data Source/60 Custom Data Sources/2 Load Data/5 Server-Side Data Processing.md (hunk 3 above)
Comment (sergepilipchuk) [reply concepts/70 Data Binding/00 Specify a Data Source/60 Custom Data Sources/2 Load Data/5 Server-Side Data Processing.md]: Done!

File: concepts/70 Data Binding/00 Specify a Data Source/60 Custom Data Sources/2 Load Data/5 Server-Side Data Processing.md (hunk 2 above)
Comment (sergepilipchuk) [reply concepts/70 Data Binding/00 Specify a Data Source/60 Custom Data Sources/2 Load Data/5 Server-Side Data Processing.md]: Done!

File: concepts/70 Data Binding/00 Specify a Data Source/60 Custom Data Sources/2 Load Data/5 Server-Side Data Processing.md (hunk 1 above)
Comment (sergepilipchuk) [reply concepts/70 Data Binding/00 Specify a Data Source/60 Custom Data Sources/2 Load Data/5 Server-Side Data Processing.md:82]: Done!

**Review by artem-kurchenko — APPROVED** [review]

**Pull request merged**